e2e/js_image/runfiles.bzl

```diff
@@ -0,0 +1,129 @@
+"contains container helper functions for js_binary"
```

Comment:
runfiles.bzl seems like a strange name, I'd go with `defs.bzl` probably since this is "public API" loaded from the BUILD.bazel file in this folder

Reply:
Agreed.